eng/testing/tests.ioslike.targets

```diff
@@ -16,17 +16,22 @@
     <!-- running aot-helix tests locally, so we can test with the same project file as CI -->
     <_AOTBuildCommand Condition="'$(ContinuousIntegrationBuild)' != 'true'">$(_AOTBuildCommand) /p:RuntimeSrcDir=$(RepoRoot) /p:RuntimeConfig=$(Configuration)</_AOTBuildCommand>
     
-    <_AOTBuildCommand>$(_AOTBuildCommand) /p:XHARNESS_EXECUTION_DIR=&quot;$XHARNESS_EXECUTION_DIR&quot; /p:RunAOTCompilation=$(RunAOTCompilation) /p:TargetOS=$(TargetOS) /p:TargetArchitecture=$(TargetArchitecture) /p:MonoForceInterpreter=$(MonoForceInterpreter) /p:MonoEnableLLVM=true /p:DevTeamProvisioning=$(DevTeamProvisioning) /p:UsePortableRuntimePack=true /p:Configuration=$(Configuration)</_AOTBuildCommand>
+    <_AOTBuildCommand>$(_AOTBuildCommand) /p:XHARNESS_EXECUTION_DIR=&quot;$XHARNESS_EXECUTION_DIR&quot; /p:RunAOTCompilation=$(RunAOTCompilation) /p:UseNativeAOTRuntime=$(UseNativeAOTRuntime) /p:TargetOS=$(TargetOS) /p:TargetArchitecture=$(TargetArchitecture) /p:MonoForceInterpreter=$(MonoForceInterpreter) /p:DevTeamProvisioning=$(DevTeamProvisioning) /p:UsePortableRuntimePack=true /p:Configuration=$(Configuration)</_AOTBuildCommand>
```

Comment:
Correct, it should be set in the project file.

Reply:
In that case, I assume `/p:MonoForceInterpreter=$(MonoForceInterpreter)` could be also removed?

Reply:
Thanks. We should verify what is happening during the build of functional tests, as in functional tests project files the property is treated with: `TreatAsLocalProperty="MonoForceInterpreter"` which afaik, means the build system will not allow overwriting `MonoForceInterpreter` value as long as it is defined in the project file - locally. 

So since `MonoForceInterpreter=false` got removed from the project file: https://github.com/dotnet/runtime/pull/87773/files#diff-67da87e0a0169c2fa25ef2d8106257412638ac6a05f4534bab3a62469fdf73aeL5 
doing `build -p:MonoForceInterpreter=true` on Helix - could build the test project differently then before.

Reply:
We can revert that if you think it may create any issues. According to the Helix jobs, the apps are properly compiled using full AOT mode.

Reply:
Looking at the other functional tests project files, most of them have `MonoForceInterpreter` set, but some have `TreatAsLocalProperty="MonoForceInterpreter"` some don't, so I believe the `TreatAsLocalProperty` was added as a workaround to prevent overwriting what a project specifies through the command line - like building on Helix.

To conclude, I feel like this needs to be refactored (maybe not in this PR), with either removing any notion of `MonoForceInterpreter` from the functional test's project file (and test the change), or leave it how it was to prevent any unwanted regression.

@steveisok what do you think?

Reply:
Here is a related change https://github.com/dotnet/runtime/pull/52606. My understanding is that it was added for local development and testing.

Reply:
All the relevant tests have passed. @steveisok What do you suggest doing here?

Reply:
@ivanpovazan Reverted the changes to be on the safe side. Please take a look again at this PR and let's try to resolve comments before the RC1 snap.